internal/pkg/config/v1/config_test.go

```diff
@@ -405,10 +416,21 @@ func TestConfig_Save(t *testing.T) {
 			if err := tt.config.Save(); (err != nil) != tt.wantErr {
 				t.Errorf("Config.Save() error = %v, wantErr %v", err, tt.wantErr)
 			}
+
 			got, _ := os.ReadFile(configFile.Name())
 			want, _ := os.ReadFile(tt.wantFile)
-			if utils.NormalizeNewLines(string(got)) != utils.NormalizeNewLines(string(want)) {
-				t.Errorf("Config.Save() = %v\n want = %v", utils.NormalizeNewLines(string(got)), utils.NormalizeNewLines(string(want)))
+			data := Config{}
+			err := json.Unmarshal(got, &data)
+			require.NoError(t, err)
+			wantString := strings.Replace(string(want), authTokenPlaceholder, data.ContextStates[contextName].AuthToken, -1)
```

Comment:
This looks ugly but is necessary.. the random salt and nonce generated is different each run. use a placeholder to replace them so they can match.